src/python/doc/ripsnet.inc

```diff
@@ -0,0 +1,15 @@
+   |                                                                | RipsNet is a general architecture for fast and robust estimation of the | :Author: Felix Hensel, Mathieu Carrière                                                                                |
+   |                                                                | persistent homology of point clouds.                                    |                                                                                                                        |
+   |                                                                |                                                                         | :Since: GUDHI                                                                                                          |
+   |                                                                |                                                                         |                                                                                                                        |
+   |                                                                |                                                                         | :License: MIT (`GPL v3 </licensing/>`_)                                                                                |
+   |                                                                |                                                                         |                                                                                                                        |
+   |                                                                |                                                                         |                                                                                                                        |
+   |                                                                |                                                                         |                                                                                                                        |
+   +----------------------------------------------------------------+-------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------+
+   | * :doc:`ripsnet`                                               |                                                                                                                                                                                                  |
+   +----------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
I don't know yet where, but at some point, we will need to `.. include:: ripsnet.inc` somewhere in the already existing documentation. To be discussed with @mglisse and @MathieuCarriere